clientSide/src/components/DropDown/DropDown.jsx

```diff
@@ -0,0 +1,54 @@
+import "./DropDown.css";
+import React, { useEffect, useState } from "react";
+import Parse from "parse";
+
+const DropDown = (props) => {
+  const [isLoading, setLoading] = useState(true);
+  const [elements, setElements] = useState(["nothing"]);
```

Comment:
A nice way to state that we need an array of strings is the following: `useState<string[]>([]);` In this case you don't need a dummy value like "nothing". But we will also use integers I guess so we could simply say `useState([]).`
Maybe useless comment, just adding useful information for the future.